README.md

```diff
@@ -29,3 +29,4 @@ a pull request to suggest improvements or add clarifications. Please use issues
     - Exit Survey
   - Learning & Development
 - Tools
+- [Products](docs/products)
```

Comment:
Considering moving Product higher in the list, (even potentially within **Company**)
- I think it is important that it is visual to all, and easy to spot